### DISCUSSION TITLE
Gateway API v1.2 Release Qualification

### DISCUSSION THREAD

**Discussion by robscott** [kubernetes-sigs/gateway-api#3349] [General]
With Gateway API [v1.2.0-rc1 released](https://github.com/kubernetes-sigs/gateway-api/releases/tag/v1.2.0-rc1), we'd like some feedback on if the RC is viable before we move on. After chatting with the other maintainers, we decided to formalize this process. We'll plan to move forward with RC2 as soon as at least one of the following has happened:

1. 3 implementations have confirmed that RC1 (or a follow up commit, such as https://github.com/kubernetes-sigs/gateway-api/pull/3343) is working well with their implementation
2. 2 weeks have passed since RC1 has been released (that would be September 30)

I'm guessing we're already pretty close to hitting 1, if we can get enough implementations to sign off, we should be able to speed up this release process.

/cc @arkodg @howardjohn @mlavacca @sunjayBhatia @youngnick

**Comment by howardjohn** [comment]
Istio has merged 1.2 (rc1 + https://github.com/kubernetes-sigs/gateway-api/pull/3343) into our `master` branch without issues

**Comment by youngnick** [comment]
Cilium has had RC1 pass our conformance just fine (although our current PR is held up with other CI issues, sigh).

**Comment by emilevauge** [comment]
[Traefik](https://github.com/traefik/traefik) passed [all conformance tests](https://github.com/traefik/traefik/blob/ae9ab9d1209925cfca42ad76ebcee6226d0ba9aa/integration/conformance-reports/v1.2.0-rc1/experimental-v3.2-default-report.yaml) with Gateway API v1.2 rc1 👌

**Comment by robscott** [comment]
Thanks to everyone that responded here! We released RC2 _yesterday_, and will plan to release v1.2.0 early next week unless we hear of any problems.
